packages/realm/RealmJS.podspec

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+require 'json'
```

Comment:
Should we remove this file from the root project?  Or will that be done later?

Reply:
Yeah. I don't know - we could do git moves instead of straight up copies. Perhaps that would ease merging in the future.

Reply:
I git moved it instead.